src/Infrastructure/Translation/TranslatableDescriptionStub.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure\Translation;
+
+use WMDE\Euro\Euro;
+use WMDE\Fundraising\Frontend\RefactoringException;
+use WMDE\Fundraising\PaymentContext\Domain\Model\PaymentInterval;
+use WMDE\Fundraising\PaymentContext\Domain\PaymentUrlGenerator\TranslatableDescription;
+
+class TranslatableDescriptionStub implements TranslatableDescription {
+
```

Comment:
```suggestion
// TODO add constructor with translation key, translations and locale
// locale could also be setter-injected, since it's determined at runtime and may not be correct at construction time 
```